frontend/src/components/regions/CardDetails/CardDetails.test.js

```diff
@@ -111,3 +111,36 @@ test(`canSetDueTime function returns false if card doesn't belong to current use
 
   expect(canSetDueTime({ card, viewedUser, authUser })).toBe(false);
 });
+
+test(`canSetDueTime function returns true if card doesn't belongs to current user, 
```

Comment:
This is complicated.

How about `canSetDueTime function returns true for superusers`